unit_tests/test_views.py

```diff
@@ -27,9 +27,10 @@ def test_index(self):
         self.assertContains(response, 'The Theater that was Rome')
 
     def test_about(self):
+        models.Static.objects.create(title='About', text='### Red Sox lineup[^n1]\n\n[^n1]: footnote text')
         response = self.client.get(reverse('about'))
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, '<h2>About</h2>')
+        self.assertContains(response, 'Red Sox')
```

Comment:
Great, tests pass now. Could you add the following lines as well? They help verify that the markdown is rendered properly.
        self.assertContains(response, '<h3>Red Sox lineup') #make sure that basic markdown was rendered
        self.assertContains(response, '<p>footnote text') #make sure that footnote was rendered

Reply:
Could you paste that into a code block? I think github formatted the markdown.

Reply:
`self.assertContains(response, 'Red Sox lineup') #make sure that basic markdown was rendered

self.assertContains(response, 'footnote text') #make sure that footnote was rendered`